pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -757,14 +757,19 @@ type PoolSpec struct {
 	// +optional
 	// +nullable
 	Quotas QuotaSpec `json:"quotas,omitempty"`
+
+	// The application name to set on the pool. Only expected to be set for rgw pools.
+	// +optional
+	// +nullable
+	Application string `json:"application"`
```

Comment:
I don't think strings are nullable since not a pointer type, but I'm not 100% certain.

```suggestion
	// The application name to set on the pool. Only expected to be set for RGW pools.
	// +optional
	Application string `json:"application"`
```

Reply:
Looks like it doesn't change the behavior of the strings, but certainly no need to label it nullable, will remove it.